modules/fuse-box-responsive-api/index.js

```diff
@@ -235,7 +235,17 @@
             /* @if codeSplitting */
             if (bMapping.i && bMapping.i[id]) {
                 var data = bMapping.i[id];
-                req(bMapping.c.b + data[0], function(err, result) {
+                /* @if universal */
+                const contextualPath = isBrowser ? bMapping.c.b : bMapping.c.s;
```

Comment:
LGTM except for `const` please, convert it to `var` and I think we can merge it.
Good job!

Reply:
Oh there is one thing. `bMapping.c.b` can it be undefined ?! Can you check that please